sound/soc/sof/intel/pci-tgl.c

```diff
@@ -39,6 +39,7 @@ static const struct sof_dev_desc tgl_desc = {
 static const struct sof_dev_desc tglh_desc = {
 	.machines               = snd_soc_acpi_intel_tgl_machines,
 	.alt_machines		= snd_soc_acpi_intel_tgl_sdw_machines,
+	.use_acpi_target_states	= true,
```

Comment:
that's a valid fix, but it needs to appear as such to maintainers.

Please reword the commit message, e.g.

ASoC: SOF: tgl: add missing use_acpi_target_states for the tgl platforms

All Intel tgl platforms should support the feature of getting the system state from acpi to deal with S0ix support.
This was missed in previous commits, likely due to copy/paste from older code.

Reply:
@plbossart Thanks for the suggestion of the commit messages.